Makefile

```diff
@@ -411,37 +419,53 @@ $(serviceapi): $$(call godeps,cmd/vic-machine-server/*.go) $(serviceapi-server)
 	@$(TIME) $(GO) build $(RACE) -ldflags "$(LDFLAGS)" -o $@ ./cmd/vic-machine-server
 
 
-$(iso-base): isos/base.sh isos/base/*.repo isos/base/isolinux/** isos/base/xorriso-options.cfg
-	@echo building iso-base docker image
-	@$(TIME) $< -c $(BIN)/.yum-cache.tgz -p $@
+# ISO Targets
+$(iso-base): isos/base.sh isos/base/repos/photon-2.0/*.repo isos/base/isolinux/** isos/base/xorriso-options.cfg
+	@echo "building iso-base image (photon-2.0)"
+	@$(TIME) $(DOCKER_BUILD) $< -r photon-2.0 -c $(BIN)/.yum-cache-photon-2.0.tgz -p $@
 
-# appliance staging - allows for caching of package install
-$(appliance-staging): isos/appliance-staging.sh $(iso-base)
-	@echo staging for VCH appliance
-	@$(TIME) $< -c $(BIN)/.yum-cache.tgz -p $(iso-base) -o $@
+$(iso-base-custom): isos/base.sh isos/base/repos/$(REPO)/*.repo isos/base/isolinux/** isos/base/xorriso-options.cfg
+	@echo "building custom iso-base image ($(REPO))"
+	@$(TIME) $(DOCKER_BUILD) -d $(REPO) $< -r $(REPO) -c $(BIN)/.yum-cache-$(REPO).tgz -p $@
```

Comment:
Below are generated cache files while you build ISO with photon-2.0 and centos-6.9.
.yum-cache-photon-2.0.tgz
.yum-cache-centos-6.9.tgz
Do you want to change them to a directory like cache-tdnf or cache-yum? I didn't get the point why "a yum-cache is compatible with a tdnf cache" is concerned because they are different cache files. If yum is confusing, I would like to change the cache file names to below:
.package-cache-photon-2.0.tgz
.package-cache-centos-6.9.tgz